deployment/CP05MOAS-GL559_Deploy.csv

```diff
@@ -1,7 +1,7 @@
-CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
-TS20210131,,,,CP05MOAS-GL559-00-ENG000000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,,,CP05MOAS-GL559-01-ADCPAM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,,,CP05MOAS-GL559-02-FLORTM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,,,CP05MOAS-GL559-03-CTDGVM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,,,CP05MOAS-GL559-04-DOSTAM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00436,39.92493,-71.1781,,1000,1000,FZ-1,,
-TS20210131,,,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1,,
\ No newline at end of file
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+TS20210131,,TS20210419,,CP05MOAS-GL559-00-ENG000000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-01-ADCPAM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-02-FLORTM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-03-CTDGVM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-04-DOSTAM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00436,39.92493,-71.1781,,1000,1000,FZ-1,,
+TS20210131,,TS20210419,,CP05MOAS-GL559-05-PARADM000,1,1,2021-01-31T18:01:00,2021-04-19T17:44:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92493,-71.1781,,1000,1000,FZ-1,,
```

Comment:
looks good; recovery cruise is in cruiseinformation.csv

Reply:
Looks good.  Cruise ID is in CruiseInformation, and date/time matches draft of P16a QLR.